apps/web/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
     "next": "~15.3.2",
-    "react": "19.0.0",
```

Comment:
I think these dependencies should only go in the root package.json if I remember correctly!